api/src/modules/document/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Operation, utils } from "document-model/document";
+import fetch from "node-fetch";
+
+export async function verifyOperationsAndSignature(operations: Operation[]) {
+  const results = await Promise.all(operations.map(async (operation) => {
```

Comment:
When we move this to the document drive we should try to reuse the hash of an operation as the previousHash for the subsequent operation

Reply:
could you help me with this?